pkg/operation/botanist/secrets.go

```diff
@@ -501,6 +501,19 @@ func (b *Botanist) generateWantedSecrets(basicAuthAPIServer *secrets.BasicAuth,
 			UsedForSSH: false,
 		},
 
+		// Secret definition for blackbox-exporter
+		&secrets.CertificateSecretConfig{
```

Comment:
Can you elaborate on how this secret is used by the blackbox-exporter? I can't find it.

Reply:
It was used in the blackbox exporter config. Could be useful in the future but since its not being used I will remove it.

Reply:
Those secrets are needed mainly for compnents not running in a `Shoot` cluster and which need certificate to authenticate. In this case a `ServiceAcccount` authentication should be sufficient.